client/.eslintrc.js

```diff
@@ -1,27 +1,61 @@
-'use strict';
-
 module.exports = {
   root: true,
   parser: 'babel-eslint',
   parserOptions: {
     ecmaVersion: 2018,
     sourceType: 'module',
     ecmaFeatures: {
-      legacyDecorators: true
-    }
+      legacyDecorators: true,
+    },
   },
   plugins: [
-    'ember'
+    'ember',
   ],
   extends: [
     'eslint:recommended',
-    'plugin:ember/recommended'
+    'plugin:ember/recommended',
+    'airbnb-base',
```

Comment:
I added this, and I should help break it up into a new PR

Reply:
https://github.com/NYCPlanning/labs-applicant-portal/pull/44